tools/code_format/check_format.py

```diff
@@ -1074,11 +1077,13 @@ def normalize_path(path):
 
     def check_visibility(error_messages):
         command = (
-            "git diff $(tools/git/last_github_commit.sh) -- source/extensions/* %s |grep '+.*visibility ='"
-            % [f"':(exclude){c}' " for c in format_checker.config["visibility_excludes"]])
+            "git diff $(tools/git/last_github_commit.sh) -- source/extensions/* %s | grep '+.*visibility ='"
+            % ''.join([f"':(exclude){c}' " for c in format_checker.config["visibility_excludes"]]))
```

Comment:
I think there was a bug in how the excludes list was joined. Previous output was:
`["':(exclude)source/extensions/early_data/BUILD' ", "':(exclude)source/extensions/filters/http/buffer/BUILD' ", ... ]`

Reply:
Fixed issue so excludes worked, fixes imports and added better logging for future clarity